docs/visitor/data/download.md

```diff
@@ -0,0 +1,45 @@
+# How to download data?
+
+You can have a look at available datasets on the [SEDOS Dashboard](https://sedos.apps.rl-institut.de/) at section "Explore the Model Structure".
+There, you can  select a model structure and see related dataset by clicking on "Look at Artifacts":
+![](../../graphics/look_at_artifacts.png)
+
+On the artifacts page you can explore the different files of the dataset and have a look at underlying data.
```

Comment:
I think here we always refer to our main SEDOS collection. This is not dependent on the selected model structure.
Actually, only the network graphs are.

Reply:
You are right - I can adapt.